### ISSUE TITLE
Network: show the port and whether TLS is used in networks

### ISSUE THREAD

**Issue by Mikaela** [ProgVal/Limnoria#1200]
Current output is:

```
2015-12-20 09:28:06+0200 -- Notice(Euforia): freenode: rajaniemi.freenode.net, ircnet: irc.nebula.fi, and pirateirc: sydney-au.pirateirc.net
```

I would be interested in knowing which port the bot is using and while TLS is seen with `/whois`, `network whois` doesn't show it and sometimes bots are in different networks than you are (e.g. relaying).

*I am hoping enough sense has been used to disable SSLv3 from Limnoria...*

**Comment by ProgVal** [comment]
You can see if it uses TLS by looking at `supybot.networks.freenode.ssl` and `supybot.networks.freenode.requirestarttls`.
Regarding the port, it wouldn't make much sense to put it here, because this command does not show the hostname of the servers but their names.

**Comment by GLolol** [comment]
I think this could be done by appending something like `:+6697` or `:6667`, since that is the same for the configured server and the actual connected one.

**Comment by ProgVal** [comment]
@GLolol Do you mean this syntax is valid in the config?

**Comment by GLolol** [comment]
No, I mean to append it to the 'networks' output: e.g. `net1: irc.somenet.abc:6667, and net2: irc.somenet.def:+6697`
